lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -831,6 +850,69 @@ static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
   return true;
 }
 
+/// Determine structural equivalence of two methodss.
+static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
+                             CXXMethodDecl *Method1, CXXMethodDecl *Method2) {
```

Comment:
The formatting is off here, use clang format.

Reply:
Reformat the whole file (it is full of such format problems)?

Reply:
Replace every redundant typename with `auto` (can be done in lot of places in the file)?

Reply:
I think only reformat/replace in the code you change. If we are about to clean up old code, it is better done in a separate pull request.